Automattic/pocket-casts-android
# Use a single icon for folders upsell

## Description

This PR removes the locked folders icon and uses an icon with the plus instead. I also cleaned up folder icons a bit by removing redundant ones and moving them to the `:images` module.

Context: p1730155873484489/1730075993.721179-slack-C05RR9P9RAT

## Testing Instructions

1. Open the app as a free user.
2. Go to the podcasts screen.
3. You should see a folders icon with a plus.
4. Tap on the icon.
5. An upsell flow should start.
6. Go back and open any podcast.
7. You should see a folder icon with a plus.
8. Tap on the icon.
9. An upsell flow should start.

## Screenshots or Screencast 

### Podcasts

| Before | After |
| - | - |
| ![glob_before](https://github.com/user-attachments/assets/361afc37-6d5f-40f2-ad4a-af5a32d51d15) | ![glob_after](https://github.com/user-attachments/assets/81823a0c-fc3d-4a6a-9304-7f696d4d1b5f) |

### Podcast

| Before | After |
| - | - |
| ![loc_before](https://github.com/user-attachments/assets/2f2e35da-ccc3-4b60-9ab3-5774bce8118c) | ![loc_after](https://github.com/user-attachments/assets/40a85a7f-a87f-4169-a964-30518efd42f7) |

## Checklist
 
#### I have tested any UI changes...
